docs/dataservice-read-catalog-example.md

```diff
@@ -30,27 +33,37 @@ To execute the example, run:
 ./examples/dataservice-read/dataservice-read-example
 ```
 
-If everything is fine, you would see `hrn:here:data:::here-optimized-map-for-visualization-2` catalog description, number of partitions and the size of `100000093` partition.
+After building and running the example, a catalog description  `hrn:here:data:::here-optimized-map-for-visualization-2`, number of partitions
+and the size of `100000093` partition display automatically.
 
 ## Building and running on Android
 
-This example shows how to integrate the HERE OLP Edge SDK C++ libraries into the Android project and shows how to read data from a catalog layer and partition.
+This example shows how to integrate the HERE OLP Edge SDK C++ libraries into the
+Android project and read data from a catalog layer and partition.
 
 ### Prerequisites
 
-* Setup the Android environment.
-* Provide correct keyId and secret key in the `example.cpp` (refer to the beginning of this README file).
+* Set up the Android environment.
+* Replace the placeholders in `examples/dataservice-read/example.cpp` with your
+app access key id and secret access key.
 
 ### Build HERE OLP Edge SDK C++
 
-First, before building, you need to configure the SDK with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`, the path to Android NDK's toolchain file set via the `CMAKE_TOOLCHAIN_FILE` variable, and, optionally, the [NDK-specific CMake variables](https://developer.android.com/ndk/guides/cmake#variables).
+Before you build the example, complete the following:
+* Configure the SDK with `EDGE_SDK_BUILD_EXAMPLES` set to `ON`.
+* Disable tests with `EDGE_SDK_ENABLE_TESTING` set to `OFF`.
+* Specify the path to Android NDK's toolchain file set via the `CMAKE_TOOLCHAIN_FILE`
+variable.
+* In case you want to build Edge SDK for a specific Android platform, use `-DANDROID_PLATFORM CMake` flag, and `-DANDROID_ABI`, if you want to build for specific Android
```

Comment:
the last two points belong together. I don't think it makes sense to separate them